vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -497,10 +505,17 @@ void Contribution::Process(
     ledger::ContributionQueuePtr queue,
     ledger::BalancePtr balance) {
   if (!queue || !balance) {
```

Comment:
Should only check queue here, since it's checking balance below.

Reply:
fixed